synapse/lib/view.py

```diff
@@ -451,6 +451,9 @@ async def storm(self, text, opts=None):
         if editformat not in ('nodeedits', 'splices', 'count', 'none'):
             raise s_exc.BadConfValu(mesg='editformat')
 
+        if editformat == 'splices':
+            s_common.deprdate('storm option editformat=splices', '2023-11-01')
```

Comment:
use `s_common. splicedeprdate constant` 

This always warns for a httpapi user.